src/ubuntu/stage-upgrade.md

```diff
@@ -1,137 +1,4 @@
 ### Stage the upgrades
```

Comment:
This seems redundant now. I think either this section or the previous subtitle `Stage all protocol upgrades` needs to be removed.

Reply:
Removed from `setup-testnet-validator-from-scratch.md` and deleted the file for now.